Formula/fossil.rb

```diff
@@ -8,7 +8,7 @@ class Fossil < Formula
 
   livecheck do
     url "https://www.fossil-scm.org/index.html/uv/download.js"
-    regex(/"title": *?"Version (\d+(?:\.\d+)+)/i)
+    regex(/"title": *?"Version (\d+(?:\.\d+)+)(?!.*preview)/i)
```

Comment:
```suggestion
    regex(/"title": *?"Version (\d+(?:\.\d+)+)\s*?\(/i)
```

It looks like they've stabilized on the `Version 2.12.1 (2020-08-20)` format for the time being, so it's probably best to simply use the opening parenthesis as a trailing delimiter. This should catch all unstable versions without having to specifically omit any new style (e.g., `RC1`, `alpha`, `beta`, etc.) that may appear in the future.

Reply:
👍 works for me (less restrictive :) )